.github/ISSUE_TEMPLATE/config.yml

```diff
@@ -0,0 +1 @@
+blank_issues_enabled: false
```

Comment:
If we're disabling blank templates, should we also add another template for features? For example, I don't think [this](https://github.com/Shopify/function-runner/issues/9) would classify as a bug.

Reply:
We can if you think it's valuable to open this repo up for feature requests

Reply:
We can also allow blank issues if you think that's more suitable

Reply:
I'd prefer to also allow the option for blank issues. Although I suppose nothing's stopping you from choosing bug report and just replacing the content with whatever you want.